MetaMask/metamask-extension
# feat: add gravity chain logo svg

# Description
Add Gravity Alpha Mainnet and Testnet svg logo

# Related issues
Fixes:

# Manual testing steps
1. Go to add network
2. Add gravity networks
  - gravity networks should have logo
 
# Screenshots/Recordings

# Pre-merge author checklist


# Pre-merge reviewer checklist
